static/js/user_groups_settings_ui.js

```diff
@@ -1,14 +1,35 @@
 import $ from "jquery";
 
+import render_browse_user_groups_list_item from "../templates/user_group_settings/browse_user_groups_list_item.hbs";
 import render_user_group_settings_overlay from "../templates/user_group_settings/user_group_settings_overlay.hbs";
 
 import * as browser_history from "./browser_history";
+import * as ListWidget from "./list_widget";
 import * as overlays from "./overlays";
+import * as people from "./people";
+import * as ui from "./ui";
+import * as user_groups from "./user_groups";
 
 export function setup_page(callback) {
     function populate_and_fill() {
+        $("#manage_groups_container").empty();
```

Comment:
Shouldn't this be added in previous commit itself?

Reply:
Yeah, and usually it's a better pattern to just do `.html()` rather than `.empty` + `.append`.